rebasehelper/utils.py

```diff
@@ -115,27 +123,33 @@ def should_use_colors(cls, conf):
         return True
 
     @classmethod
-    def cprint(cls, message, color=None):
-        """
-        Print colored output if possible
+    def cprint(cls, message, fg=None, bg=None, style=None):
+        """Prints colored output if possible.
+
+        Args:
+            message (str): String to be printed out.
+            fg (str): Color to be used as a foreground in the printed message.
+            bg (str): Color to be used as a background in the printed message.
+            style (str): Style to be applied to the printed message.
 
-        :param color: color to be used in the output
-        :param message: string to be printed out
         """
-        if cls.use_colors and color is not None and hasattr(colors, color):
-            print(getattr(colors, color)(message))
+        if cls.use_colors:
+            print(colors.color(message, fg=fg, bg=bg, style=style))
         else:
             print(message)
 
     @staticmethod
     def get_message(message, default_yes=True, any_input=False):
-        """
-        Function for command line messages
+        """Prompts a user with yes/no message and gets the response
```

Comment:
`an user`.

Reply:
And missing dot at the end.

Reply:
As far as I know it depends on the pronunciation in this case, therefore it should be a user (https://ell.stackexchange.com/a/26988)

Reply:
Never realized that, thanks. I guess I've seen `an user` written so many times that it didn't even occur to me that it could be wrong.